python_modules/dagster-graphql/dagster_graphql/schema/roots/mutation.py

```diff
@@ -316,6 +317,35 @@ def mutate(
         return create_execution_params_and_launch_pipeline_exec(graphene_info, executionParams)
 
 
+class GrapheneLaunchMultipleRunsMutation(graphene.Mutation):
+    """Launches multiple job runs."""
+
+    Output = graphene.NonNull(GrapheneLaunchMultipleRunsResult)
+
+    class Arguments:
+        executionParamsList = non_null_list(GrapheneExecutionParams)
+
+    class Meta:
+        name = "LaunchMultipleRunsMutation"
+
+    @capture_error
+    @require_permission_check(Permissions.LAUNCH_PIPELINE_EXECUTION)
```

Comment:
This permission check here can make the whole mutation fail, and in this case the output would be a single GrapheneUnauthorizedError so I believe your output should be a union of `GrapheneLaunchMultipleRunsResult` and `GrapheneUnauthorizedError` and also `GraphenePythonError` since we're using `@capture_error`.